src/Dom/Document/Encoding.php

```diff
@@ -38,6 +38,9 @@ interface Encoding
     const MAPPINGS = [
         // Assume ISO-8859-1 for some charsets.
         'latin-1' => 'ISO-8859-1',
+        // US-ASCII is one of the most popular ASCII names and used as HTML charset,
+        // but mb_list_encodings contains only "ASCII"
```

Comment:
This library originated from within a WordPress plugin, so we have some consistency rules that come from the WordPress Coding Standards (WPCS). One of these rules is that all comments are terminated with a period. We're keeping this rule around for consistency.
(And I now notice that we are not actually checking these rules atm...)
```suggestion
        // but mb_list_encodings contains only "ASCII".
```

Reply:
I created a new issue for this, as ideally all such rules should be enforced via code quality checks: #184

Reply:
Corrected the comment.